tensorflow/compiler/xla/window_util.h

```diff
@@ -55,6 +55,10 @@ bool HasBaseDilation(const Window& window);
 bool HasWindowDilation(const Window& window);
 bool HasDilation(const Window& window);
 
+// Return true if the window overlap (e.g., the window size is smaller
```

Comment:
"Returns", "overlaps", I would remove "e.g.," in brackets since that's how the function is defined.

Reply:
Fixed